portfolio/src/main/webapp/js/comments.js

```diff
@@ -57,18 +65,19 @@ function deleteComment(commentID) {
     params.append('action', 'delete');
     fetch('/comments', {method: 'POST', body: params}).then(response => {
         if (response.status === 200)
-            getComments(selectedSortType);
+            getComments(selectedSortType, selectedQuantityOfComments);
         else
             console.error(response.message);
     });
 }
 
-if (selectedSortType === 'date' || selectedSortType === 'rating')
-    getComments(selectedSortType);
-else {
-    getComments('date');
+if (selectedSortType !== 'date' && selectedSortType !== 'rating')
     selectedSortType = 'date';
-}
+
+if (selectedQuantityOfComments === null || selectedQuantityOfComments === undefined)
```

Comment:
I think `selectedQuantityOfComments == null` will return true for both `=== null` and `=== undefined`